Shared/Sources/DataSource.swift

```diff
@@ -8,8 +8,8 @@ final class DataSource<Object: Collectable, Cell: UITableViewCell>: NSObject, UI
   typealias FooterTitleRetriever = ((CollectableGroup<Object>) -> String?)?
 
   required init(collection: CollectableCollection<Object>, configureCell: CellConfigurator, cellIdentifier: String, titleForFooter: FooterTitleRetriever = nil) {
+    self.completeCollection = collection
```

Comment:
I'd apply the same comment here as above between `filteredCollection` - but it seem that it's how it used to be. Any particular reason for that change?